### ISSUE TITLE
Breaking changes in 1.3

### ISSUE THREAD

**Issue by kevin-brown** [mfogel/django-timezone-field#27]
We just noticed a new failing build as of last night and tracked down the issue to a change in the default value for `null` in this field. It used to default to `null=True`, but now it is defaulting to `null=False` which is causing some of our validation to fail.

Running django-timezone-field @ https://github.com/mfogel/django-timezone-field/commit/04566d136c0f93c92f88acc0cfd862a7016e56f4

```python
Python 3.4.1 (v3.4.1:c0e311e010fc, May 18 2014, 10:38:22) [MSC v.1600 32 bit (Intel)] on win32
Type "help", "copyright", "credits" or "license" for more information.
(InteractiveConsole)
>>> from timezone_field import fields
>>> fields.TimeZoneField()
<timezone_field.fields.TimeZoneField>
>>> fields.TimeZoneField().null
True
>>> exit()
```

Running django-timezone-field @ https://github.com/mfogel/django-timezone-field/commit/80ab1ec489cac369a221c78f91c6401edc7f3549

```python
Python 3.4.1 (v3.4.1:c0e311e010fc, May 18 2014, 10:38:22) [MSC v.1600 32 bit (Intel)] on win32
Type "help", "copyright", "credits" or "license" for more information.
(InteractiveConsole)
>>> from timezone_field import fields
>>> fields.TimeZoneField().null
False
>>> exit()
```

This appears to have been caused by https://github.com/mfogel/django-timezone-field/commit/71f107e1ceb491b685033e5a538d24d6c7b50a1b, which suggests that the previous default should have been `null=True` and this counts as a bugfix. If you check [line 61](https://github.com/mfogel/django-timezone-field/commit/71f107e1ceb491b685033e5a538d24d6c7b50a1b#diff-1aad82537593c5af3aa6d7f355cc72c5L61) of that diff, you'll notice that the logic used to explicitly specify `True` as the default. This may also be related to https://github.com/mfogel/django-timezone-field/issues/23, which inspired the commit before that one.

We've fixed this by explicitly specifying `null=True` (which we know isn't optimal for a `CharField`) for now.

**Comment by mfogel** [comment]
Thanks for the investigation - I'll follow up ASAP and prepare a hotfix release as necessary.

**Comment by mfogel** [comment]
Ok - I see the change in behavior between those two versions. However, looking at @ what's tagged 1.2 (8be692c0):

```python
Python 2.7.10 (default, Oct 10 2015, 02:50:18)
Type "copyright", "credits" or "license" for more information.

IPython 4.0.0 -- An enhanced Interactive Python.
?         -> Introduction and overview of IPython's features.
%quickref -> Quick reference.
help      -> Python's own help system.
object?   -> Details about 'object', use 'object??' for extra details.

In [1]: from timezone_field import fields

In [2]: fields.TimeZoneField().null
Out[2]: False
```

I get the same behavior as is currently happening in 1.3 As such... I think I would view this as a bug introduced in https://github.com/mfogel/django-timezone-field/commit/ffe790a8bc5c376aeed44a2e2e7192b604dcd15c (which was committed after the 1.2 release) but then then fixed before the 1.3 release. Thoughts?

**Comment by kevin-brown** [comment]
> I get the same behavior as is currently happening in 1.3 As such... I think I would view this as a bug introduced in ffe790a (which was committed after the 1.2 release) but then then fixed before the 1.3 release. Thoughts?

And that's what I get for working from https://github.com/mfogel/django-timezone-field/commit/04566d136c0f93c92f88acc0cfd862a7016e56f4 locally instead of https://github.com/mfogel/django-timezone-field/commit/d1af4309b3fe60863a8dfb539551c76df8048d38. Needed to move a few commits further when we upgraded to 1.8.

Now things are starting to make more sense, and this isn't actually a regression from 1.2 to 1.3. Thanks for following up, it looks like I jumped a bit too quickly on this one.
